### PR TITLE
Fix consistency on terms used (users/players)

### PR DESCRIPTION
**Pull request by mangomizer**
Add consistency to the use of "Users" and "players/participants"

user = user
player/participant = a user who is confirmed playing in the tournament.

### CODE REVIEW

File: wiki/Tournaments/Official_support/en.md [hunk 1]
```diff
@@ -91,28 +91,27 @@ At the account support team's discretion, offending players may be issued timed
...
 
 The comma-separated list should look like this:
 
 ```csv
-Player1,1234567
-Player2,1234567
-Player3,1234567
+User1,1234567
+User2,1234567
+User3,1234567
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:103]: conflicts with https://github.com/ppy/osu-wiki/pull/6696

File: wiki/Tournaments/Official_support/en.md [hunk 2]
```diff
@@ -91,28 +91,27 @@ At the account support team's discretion, offending players may be issued timed
...
 For team-based tournaments, the expected format is:
 
 ```csv
-Player1,Team1,1234567
-Player2,Team1,1234567
-Player3,Team2,1234567
-Player4,Team2,1234567
+User1,Team1,1234567
+User2,Team2,1234567
+User3,Team3,1234567
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:111]: i mean this one

Comment (mangomizer) [reply wiki/Tournaments/Official_support/en.md:111]: Go with Leo's Format but replace Player with User.

File: wiki/Tournaments/Official_support/en.md [hunk 3]
```diff
@@ -181,7 +181,9 @@ We have historically had issues where unscrupulous organisers have geared tourna
 
 ### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems? {#player-join-staff}
 
-No. Participants who have been eliminated from the tournament/contest may not be enlisted as staff or assist in any organisational capacity **with the sole exception** of setting replays for mappool showcases. They can, however, be enlisted as a helper.
+No. Players who have been eliminated from the tournament/contest may not be enlisted as staff or assist in any organisational capacity with the sole exception of setting replays for mappool showcases. They can, however, be enlisted as a helper.
+
+A user who has passed screening but has not participated in any gameplay (including qualifiers) **AND** has not been scheduled for any matches does not qualify as a player/participant in our eyes, and therefore may be enlisted as staff (they have not been "eliminated" per se).
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:186]: if a non-participant hasn't played any matches, isn't it irrelevant (and not applicable) whether they have been eliminated?

if i understand this right, a user can participate as staff and later schedule matches to play in the tournament, but not later be part of staff once eliminated as a player, correct?

```suggestion
A user who has passed screening but has not participated in any gameplay (including qualifiers) **AND** has not been scheduled for any matches does not qualify as a player/participant in our eyes, and therefore may be enlisted as staff.
```

Comment (Walavouchey) [reply wiki/Tournaments/Official_support/en.md:186]:
```suggestion
A registrant who has passed screening but has not been scheduled for any matches **AND** has not participated in any gameplay (including qualifiers)matches is not yet a player of the tournament, and therefore may be enlisted as staff instead.
```

Comment (mangomizer) [reply wiki/Tournaments/Official_support/en.md:186]:
```suggestion
A registrant who has passed screening but has not been scheduled for any matches **AND** has not participated in any gameplay (including qualifiers) is not yet considered a player/participant of the tournament, and may therefore still be enlisted as staff.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**